src/main/java/com/elytradev/concrete/inventory/gui/widget/Widget.java

```diff
@@ -33,9 +33,8 @@
 import net.minecraftforge.fml.relauncher.Side;
 import net.minecraftforge.fml.relauncher.SideOnly;
 
-public class WWidget {
+public class Widget {
```

Comment:
Personally, it feels wrong to name the base widget class as `WWidget` because that means it's actually `WidgetWidget`, but if @falkreon prefers it I'll rename it back to `WWidget`.